ibis/backends/pandas/execution/__init__.py

```diff
@@ -0,0 +1,11 @@
+from .arrays import *  # noqa: F401,F403
+from .decimal import *  # noqa: F401,F403
+from .generic import *  # noqa: F401,F403
+from .join import *  # noqa: F401,F403
```

Comment:
instead of adding the noqa

add

```
__all__ = ('decimal', 'join', .....)
```

Reply:
I tried to do that but got test failures, what exactly do you mean?

I replaced all the lines in that file by __all__ = (...) but that gives failures in the tests

Reply:
like this: https://github.com/pandas-dev/pandas/blob/master/pandas/core/arrays/__init__.py

Reply:
I think I did what you have asked for, let me know if that works.

Out of curiosity, why do you want to do that? Is it for speed?

Reply:
Sorry I think I tried doing what you suggested then I get the following errors https://github.com/ibis-project/ibis/pull/2440#issuecomment-705865599

Reply:
i c, this is not just a list of functions, but much more, we should not do this. 

instead we should figure out what is actually in the public api. ok remove the ``__all__`` and pls create an isseu to followup on this.

Reply:
I have made a new issue https://github.com/ibis-project/ibis/issues/2460 and removed `__all__` for now.